### PR TITLE
fix: require bluemix

### PR DESCRIPTION
**Pull request by sydney-ng**

### CODE REVIEW

**Comment by jkingoliver** [comment]
@sydney-ng Does the readme need to be updated as well? Can you make sure the readme is accurate please?

File: app/index.js [hunk 1]
```diff
@@ -73,8 +73,8 @@ module.exports = class extends Generator {
   initializing() {
     this.skipPrompt = true;
     let bluemix_ok= this._sanitizeOption(this.options, OPTION_BLUEMIX);
-    let spec_ok= this._sanitizeOption(this.options, OPTION_SPEC);
-    if ( ! (bluemix_ok || spec_ok )) throw ("Must specify either bluemix or spec parameter");
+    //let spec_ok= this._sanitizeOption(this.options, OPTION_SPEC);
```
Comment (jkingoliver) [inline app/index.js]: can you remove this comment please?

Comment (sydney-ng) [reply app/index.js]: yes!

**Comment by jkingoliver** [comment]
This PR was applied in https://github.com/ibm-developer/generator-ibm-core-node-express/pull/146

**Pull request closed without merging**